common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/AppStarterWebSecurityAutoConfigurationProperties.java

```diff
@@ -18,21 +18,21 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
- * {@code SecurityCommonAutoConfiguration} properties.
+ * {@code DefaultAppsWebSecurityAutoConfiguration} properties.
  *
  * @author Christian Tzolov
  * @author Artem Bilan
  *
  * @since 2.1
  */
 @ConfigurationProperties("spring.cloud.stream.security")
```

Comment:
I am not sure if we can use `spring.cloud.stream` prefix for these properties.

Reply:
How about `spring.cloud.stream.app` as a prefix?

Reply:
The consensus is to use `spring.cloud.app-starter-stream.security`

Reply:
Given this property only fits for all the stream app starters (out of the box apps) and this is one of the common properties, the idea was to use `spring.cloud.stream-app-starters` and avoid the `spring.cloud.stream` property prefix so that it won't conflict with the SCSt properties.  

If `starters` brings some confusion, we can use something along the lines of `spring.cloud.stream-app-common`.